requirements.txt

```diff
@@ -1 +1 @@
-Cython==3.0a10
\ No newline at end of file
+Cython==3.0.9
```

Comment:
There's no special reason for picking this version. It was the latest when I began working on the PR. This could be bumped to 3.0.10 if desired: https://cython.readthedocs.io/en/stable/src/changes.html.